packages/mobile/src/screens/sign-up-screen/screens/PickHandleScreen.tsx

```diff
@@ -40,19 +89,63 @@ export const PickHandleScreen = () => {
     [dispatch, navigation]
   )
 
+  const handleCompleteSocialMediaLogin = useCallback(
+    ({
+      requiresReview,
+      handle,
+      platform
+    }: {
+      requiresReview: boolean
+      handle: string
+      platform: 'twitter' | 'instagram' | 'tiktok'
+    }) => {
+      dispatch(setValueField('handle', handle))
+      // TODO: social media handle impl
+      if (!requiresReview) {
+        // navigate(SIGN_UP_FINISH_PROFILE_PAGE)
+      } else {
+        // navigate(SIGN_UP_REVIEW_HANDLE_PAGE)
+      }
+      // toast(socialMediaMessages.socialMediaLoginSucess(platform))
+    },
+    [dispatch]
+  )
+
   return (
-    <View>
-      <Text>{messages.header}</Text>
-      <Text>{messages.description}</Text>
-
-      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-        {({ handleSubmit }) => (
-          <View>
-            <TextField name='handle' label={messages.handle} />
-            <Button title={messages.continue} onPress={() => handleSubmit()} />
-          </View>
-        )}
-      </Formik>
-    </View>
+    <Formik
+      initialValues={initialValues}
+      onSubmit={handleSubmit}
+      validationSchema={validationSchema}
+    >
+      {({ handleSubmit: triggerSubmit, dirty, isValid }) => (
+        <Page>
+          <Heading
+            heading={messages.title}
+            description={messages.description}
+          />
+          <Flex direction='column' gap='l'>
+            <TextField name='handle' label={messages.handle} noGutter />
```

Comment:
Can you add a TODO here for adding the verification flow error message to the handle field component? (Can see web's `HandleField`)